### DISCUSSION TITLE
Support for increasing scope of automated verification

### DISCUSSION THREAD

**Discussion by dkkapur** [filecoin-project/notary-governance#235] [Ideas]
[verify.glif.io](verify.glif.io) is currently the first and only on-chain Notary in the Fil+ System. It's current status is:
- 32GiB of DataCap granted to clients who have a GitHub account older than 180d and with no previous allocations granted in the last 30d (up from initial 8GiB)
- ~13TiB of DataCap granted to 483 client addresses (several with repeat allocations, including 2 clients with 4 allocations); this represents ~65% of all client addresses receiving DataCap on-chain
- Clients have used this DataCap in deals with 145 miners - here is the allocation breakdown: https://filplus.d.interplanetary.one/notaries/f0121877/breakdown
- DataCap granted in < 1min (~1 epoch) on average, compared to TTD of 11d+ for manual verification

As we work towards drastically increasing the funnel and improving the client onboarding path to Filecoin via Fil+/DataCap, automated verification will become an incredibly important tool for clients and notaries to unblock clients as well as increase the amount of data available on client operations on chain for future notary actions. 

Kicking off this discussion to talk about ways in which we can:
- grow confidence in automated verification to be able to support 1TiB automatic DataCap allocation in the next 6mo
- enable automated verification for clients that may not be GitHub users / from regions where other tools or forms of KYC are preferable 
- increase the presence of automated verification, whether it is through improving the presence of the current one or through RPFs for new auto-verifier notaries

**Comment by dkkapur** [comment]
Some other relevant stats:
- Average DataCap allocation size today is around 1.4TiB/client => moving to support something closer to this with automation could greatly improve the client onboarding experience to Filecoin
- Clients coming through the automated Notary make deals with >60%+ participating storage providers in the Fil+ ecosystem
- Average verified deal size is approx. 16GB; 32GB enables 2 deals which is not enough for a client to demonstrate their good use of DataCap or make real use of the network, especially given that we are often recommending clients store 5+ replicas of real data on the Network

<img width="879" alt="Screen Shot 2021-09-27 at 5 09 00 PM" src="https://user-images.githubusercontent.com/2343218/135001598-70ae02b7-88ab-4c1d-bf23-cc662b440529.png">

**Comment by de-authority** [comment]
I think we should consider lowering the length requirement for github accounts, giving them less datacap, so that they can experience our features, for example, 16G datacap for 90-180 days accounts, and only 8G datacap for accounts below 90 days. 30 days can only be applied once to prevent abuse

**Reply by galen-mcandrew** [reply]
Interesting idea! I think the ramping levels based on account age could be really useful. I wonder if there should be a lower limit, say 1 week old accounts? Or maybe some threshold of GitHub activity, like X number of interactions?
